scripts/public_api_report.py

```diff
@@ -784,6 +854,394 @@ class CodeReferenceFilter:
+            filepath=pathlib.Path(
+                "great_expectations/rule_based_profiler/helpers/cardinality_checker.py"
+            ),
+        ),
+        IncludeExcludeDefinition(
+            reason="to_json_dict is an internal helper method",
+            name="to_json_dict",
+            filepath=pathlib.Path(
+                "great_expectations/rule_based_profiler/helpers/configuration_reconciliation.py"
+            ),
+        ),
+        IncludeExcludeDefinition(
+            reason="to_json_dict is an internal helper method",
+            name="to_json_dict",
+            filepath=pathlib.Path(
+                "great_expectations/rule_based_profiler/helpers/runtime_environment.py"
+            ),
+        ),
+        IncludeExcludeDefinition(
+            reason="to_json_dict is an internal helper method",
+            name="to_json_dict",
+            filepath=pathlib.Path(
+                "great_expectations/rule_based_profiler/parameter_container.py"
+            ),
+        ),
+        IncludeExcludeDefinition(
+            reason="to_json_dict is an internal helper method",
+            name="to_json_dict",
+            filepath=pathlib.Path(
+                "great_expectations/rule_based_profiler/rule/rule.py"
+            ),
+        ),
+        IncludeExcludeDefinition(
+            reason="to_json_dict is an internal helper method",
+            name="to_json_dict",
+            filepath=pathlib.Path(
+                "great_expectations/rule_based_profiler/rule_based_profiler.py"
+            ),
+        ),
+        IncludeExcludeDefinition(
+            reason="to_json_dict is an internal helper method",
+            name="to_json_dict",
+            filepath=pathlib.Path(
+                "great_expectations/rule_based_profiler/rule_based_profiler_result.py"
+            ),
+        ),
+        IncludeExcludeDefinition(
+            reason="to_json_dict is an internal helper method",
+            name="to_json_dict",
+            filepath=pathlib.Path("great_expectations/types/attributes.py"),
+        ),
+        IncludeExcludeDefinition(
+            reason="to_json_dict is an internal helper method",
+            name="to_json_dict",
+            filepath=pathlib.Path("great_expectations/types/base.py"),
+        ),
+        IncludeExcludeDefinition(
+            reason="Internal helper method",
+            name="filter_properties_dict",
+            filepath=pathlib.Path("great_expectations/util.py"),
+        ),
+        IncludeExcludeDefinition(
+            reason="to_json_dict is an internal helper method",
+            name="to_json_dict",
+            filepath=pathlib.Path("great_expectations/validator/exception_info.py"),
+        ),
+        IncludeExcludeDefinition(
+            reason="to_json_dict is an internal helper method",
+            name="to_json_dict",
+            filepath=pathlib.Path(
+                "great_expectations/validator/metric_configuration.py"
+            ),
+        ),
```

Comment:
See above.

Reply:
Updated (not all, see https://github.com/great-expectations/great_expectations/pull/6803/commits/5be5c7016e942973c8db0fa61ac2350b293ff0a7 for specifics)